mapper/mapper_test.go

```diff
@@ -14,25 +12,33 @@ import (
 	"github.com/Financial-Times/message-queue-gonsumer/consumer"
```

Comment:
Please organize the imports the following way https://github.com/golang/go/wiki/CodeReviewComments#imports.